GVFS/GVFS.Common/GVFSConstants.cs

```diff
@@ -210,7 +210,8 @@ public static class VerbParameters
         {
             public static class Mount
             {
-                public const string ServiceName = "internal_use_only_service_name";
+                public const string InternalUseOnly = "internal_use_only";
+                public const string StartedByService = "StartedByService";

```

Comment:
I don't think this is being used any more?

Reply:
@sanoursa  Where would you recommend adding the StartedByService Constant?
It's used in both InProcessMountVerb (GVFS.Win) and MountVerb (GVFS.Mount.Win)